obspy/taup/tau.py

```diff
@@ -4,6 +4,12 @@
                         unicode_literals)
 from future.builtins import *  # NOQA
 
+import copy
+
+import matplotlib.cbook
+import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
Probably because I was thinking of SciPy and forgot they'd have put scipy in the first-party location. :stuck_out_tongue: